<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageLoader</class>
 <widget class="QWidget" name="ImageLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1929</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="slider_xy_currentLayer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>22</width>
     <height>481</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tiefenKarte">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiefenkarte</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_xy_start">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1024</number>
   </property>
   <property name="maximum">
    <number>3071</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>-400</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_xy_width">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
   <property name="value">
    <number>900</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reflect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>3D-Reflection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xy_width">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xy_currentLayer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_layered">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load 3D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xy_image">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_slicerCoordinate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(0,0,0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_point1Coordinate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(0,0,0)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_point2Coordinate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(0,0,0)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_xy_start">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>540</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start:</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_xy_threshold">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
   <property name="value">
    <number>900</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_xy_threshold">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>600</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xz_width">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>570</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xz_currentLayer">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>490</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer:</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_xz_currentLayer">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>10</y>
     <width>22</width>
     <height>481</height>
    </rect>
   </property>
   <property name="maximum">
    <number>511</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_xz_threshold">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>600</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xz_image">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>150</y>
     <width>512</width>
     <height>256</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="slider_xz_threshold">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>600</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
   <property name="value">
    <number>900</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_xz_start">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>540</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1024</number>
   </property>
   <property name="maximum">
    <number>3071</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>-400</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_xz_start">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>540</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start:</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_xz_width">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>570</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
   <property name="value">
    <number>900</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_slice_start">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>540</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slice_threshold">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>600</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold:</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_slice_threshold">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>600</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
   <property name="value">
    <number>900</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_slice_currentLayer">
   <property name="geometry">
    <rect>
     <x>1840</x>
     <y>10</y>
     <width>22</width>
     <height>481</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_slice_width">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>570</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_slice_width">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>570</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
   <property name="value">
    <number>900</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_slice_currentLayer">
   <property name="geometry">
    <rect>
     <x>1840</x>
     <y>490</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slice_image">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>10</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="slider_slice_start">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>540</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1024</number>
   </property>
   <property name="maximum">
    <number>3071</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>-400</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_point1Coordinate_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Point1:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_point2Coordinate_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Point2:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_slicerCoordinate_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Slicer:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
